app/models/clubhouse.rb

```diff
@@ -0,0 +1,36 @@
+class Clubhouse < ApplicationRecord
+  has_many :posts, dependent: :destroy
+  has_many :memberships, dependent: :destroy
+
+  validates :title, presence: true
+
+  def self.with_owner(owner, properties)
```

Comment:
I suggest you keep here the `create` word in the method name because you are creating the records and not just initializing them.

```suggestion
  def self.create_with_owner(owner, properties)
```

Reply:
Done!